cdap-data-fabric/src/main/java/co/cask/cdap/data2/util/hbase/HTableNameConverterFactory.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright © 2015 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.data2.util.hbase;
+
+/**
+ * Factory for HBase version-specific {@link HTableNameConverterFactory} instances.
+ */
+public class HTableNameConverterFactory extends HBaseVersionSpecificFactory<HTableNameConverter> {
+  @Override
+  protected String getHBase94Classname() {
+    return "co.cask.cdap.data2.util.hbase.HBase94TableUtil";
```

Comment:
That is my mistake. I didn't run all the QueueTests yet after this change, so thats why it didn't surface so far.